tests/unit/tasks/test_python_task.py

```diff
@@ -0,0 +1,26 @@
+from griptape.artifacts import BaseArtifact, TextArtifact
+from griptape.tasks import PythonTask, BaseTextInputTask, BaseTask
+from tests.mocks.mock_prompt_driver import MockPromptDriver
+from griptape.structures import Pipeline
+
+
+def hello_world(task: BaseTask) -> BaseArtifact:
+    greeting = "Hello World!"
+    # We don't want to create an artifact, so just pass the parent task's input back
+    return TextArtifact(greeting)
+
+
+def deliberate_exception(task: BaseTask) -> BaseArtifact:
+    raise ValueError("ugh blub")
```

Comment:
I will go on record and admit that my suggestion of "ugh blugh" was unprofessional and disrespectful and we should change this :)